docs/zh/docs/advanced/custom-response.md

```diff
@@ -0,0 +1,212 @@
+# 自定义响应 - HTML，流，文件和其他
+
+**FastAPI** 默认会使用 `JSONResponse` 返回响应。
+
+你可以通过直接返回 `Response` 来重载它，参见 [直接返回响应](response-directly.md){.internal-link target=_blank}。
+
+但你可以直接返回 `Response`，返回数据不会自动转换，也不会自动生成文档（例如，在 HTTP 头 `Content-Type` 中包含特定的「媒体类型」作为生成的 OpenAPI 的一部分）。
+
+你还可以在 *路径操作装饰器* 中声明你想用的 `Response`。
+
+你从 *路径操作函数* 中返回的内容将被放在该 `Response` 中。
+
+并且如果该 `Response` 有一个 JSON 媒体类型（`application/json`），比如使用 `JSONResponse` 或者 `UJSONResponse` 的时候，你返回的数据会自动转换成任何你在 *路径操作装饰器* 中声明的 Pydantic 的 `response_model` 类型。
```

Comment:
Missing words "(and filtered) ".And "converted with" might means "using Pydantic response_model to converte" not "convert to"